i18n/en.ts

```diff
@@ -384,6 +384,58 @@ Saturdays & Holidays: 09.00 am to 05.00 pm`,
     },
     sections: { clubs: { title: 'CLUBS', more: 'Explore all clubs' } },
   },
+  Unauthorised: {
+    title: 'Unauthorised',
+    description: 'You are not authorised to view this page.',
+  },
+  WorkInProgress: {
+    title: 'Work In Progress',
+    description:
+      'This page is currently being worked on. Please visit another time',
+  },
+
+  infrastructure: {
+    heading: `Campus and Infrastructure`,
+    headings: [`Campus`, `Infrastructure`, `How to reach`],
+    campus: [
+      `Campus`,
```

Comment:
I suggested a list in the previous PR because it was useful there. It doesn't work everywhere, please make these normal objects. the descriptions can still be a list but you would logically separate out the title